File: lista_Logica_v2/ex2.py
# 2) A prefeitura de uma cidade fez uma pesquisa entre seus habitantes, coletando dados sobre o salário e número de filhos. A prefeitura deseja saber:  
    # a) média do salário da população;
    # b) média do número de filhos;
    # c) maior salário;
    # d) percentual de pessoas com salário até R$1000,00.
# Faça um software que permita a coleta desses dados e ao final, imprima na tela as informações desejadas.

sal = 0
SomaSal = 0
maiorSal = 0.0
filhos = 0
continua = 1
cont = 0
ateMil = 0
while (continua != 0):
    sal = float(input(f"Digite o salario da família {cont + 1}: "))
    filhos += int(input(f"Digite quantos filhos/as tem a família {cont + 1}: "))
    if (sal > maiorSal):
        maiorSal = sal
    if (sal <= 1000.00):
        ateMil = ateMil+1
    SomaSal += sal
    cont += 1
    continua = int(input("Digite 0 para sair ou 1 para continuar: "))

print(f"A média do salário da população é {(SomaSal / cont):.2f}")
print(f"A média do número de filhos é {(filhos / cont):.2f}")
print(f"O maior salário é {maiorSal:.2f}")
print(f"O percentual de pessoas com salário até R$ 1000,00 é de {((ateMil*100) / cont):.2f} %")

File: lista_Logica_v2/emC/ex2.c
#include <stdio.h>

void main() {
    float sal = 0;
    float somaSal = 0;
    float maiorSal = 0.0;
    int filhos = 0;
    int continua = 1;
    int cont = 0;
    int ateMil = 0;
    int filhos_familia;

    while (continua != 0) {
        printf("Digite o salario da familia %d: ", cont + 1);
        scanf("%f", &sal);
        
        printf("Digite quantos filhos/as tem a familia %d: ", cont + 1);

        scanf("%d", &filhos_familia);
        filhos += filhos_familia;
        
        if (sal > maiorSal) {
            maiorSal = sal;
        }
        
        if (sal <= 1000.00) {
            ateMil = ateMil + 1;
        }
        
        somaSal += sal;
        cont += 1;
        
        printf("Digite 0 para sair ou 1 para continuar: ");
        scanf("%d", &continua);
    }
    
    printf("A media do salario da populacao e %.2f\n", (somaSal / cont));
    printf("A media do numero de filhos e %.2f\n", ((float)filhos / cont));
    printf("O maior salario e %.2f\n", maiorSal);
    printf("O percentual de pessoas com salario ate R$ 1000,00 e de %.2f %%\n", ((ateMil * 100.0) / cont));

}